consensus/istanbul/backend/announce.go

```diff
@@ -114,9 +120,7 @@ func (sb *Backend) announceThread() {
 				// hence more likely that they will be aware that this node is
 				// within that set.
 				time.AfterFunc(1*time.Minute, func() {
-					if err := sb.generateAndGossipAnnounce(announceVersion); err != nil {
-						logger.Error("Error in gossiping announce", "err", err)
-					}
+					sb.startGossipAnnounceTask()
 				})
 
 				if sb.config.AnnounceAggressiveGossipOnEnablement {
```

Comment:
Nit: Comment on line 138 is a bit misleading now that the announcing variable exists